3.train_ebimage/1.random_forest/random-forest-pipeline.sh

```diff
@@ -3,15 +3,4 @@
 # Process the data using methods centered around random forest classifiers
 set -e
 
-# Convert notebooks to .py
-jupyter nbconvert --to=python \
-  --FilesWriter.build_directory=scripts/nbconverted \
-  *.ipynb
-
-# Run the notebook
-jupyter nbconvert --to=html \
-  --FilesWriter.build_directory=scripts/html \
-  --ExecutePreprocessor.kernel_name=python \
-  --ExecutePreprocessor.timeout=10000000 \
-  --execute random_forest.ipynb
-  
+python ./3.train_ebimage/1.random_forest/random-forest.py
```

Comment:
```suggestion
python ./3.train_ebimage/1.random_forest/random-forest.py

```

Adding a new line. Shown is that weird [no new line octicon](https://octicons.github.com/icon/no-newline/). Adding a new line removes it